kart/__init__.py

```diff
@@ -199,7 +199,10 @@ def _cleanup_process_group(signum, stack_frame):
             if _kart_process_group_killed:
                 return
             _kart_process_group_killed = True
-            os.killpg(os.getpid(), signum)
+            try:
+                os.killpg(os.getpid(), signum)
+            except Exception:
```

Comment:
can you explain this change? I don't think I get it

Reply:
> but maybe since it was already being killed, it was no longer findable?

Is `killpg(mypid)` any different from `killpg(0)`?